### ISSUE TITLE
Can Support Chinese？

### ISSUE THREAD

**Issue by wsh2836741** [egerber/spaCy-entity-linker#35]
Hi，thanks for your excellent job，and can it support chinese entity linking？

**Comment by dpriskorn** [comment]
No, it's hard coded to English right now.

**Comment by ottowg** [comment]
Do you think it could be possible to implement multi language support?

I am interested in German language support and are also interested in investing coding time on this.

Can you summarize, which components need to be touched for that?
